src/producer.ts

```diff
@@ -76,6 +70,10 @@ export class Producer {
     )
   }
 
+  public on(_eventName: "publish_confirm", cb: PublishConfirmCallback) {
+    this.connection.on("publish_confirm", (confirm: PublishConfirmResponse) => cb(null, confirm.publishingIds))
```

Comment:
Why is it null error??

Reply:
It should be because if there is no error we call the callback with null while we call the callback with the errorcode if there is an error